liquibase-snowflake/src/main/java/liquibase/sqlgenerator/core/InsertOrUpdateGeneratorSnowflake.java

```diff
@@ -63,7 +63,7 @@ protected String getElse(Database database) {
     // Copied from liquibase.sqlgenerator.core.InsertOrUpdateGeneratorHsql
     private String convertToString(Object newValue, Database database) {
         String sqlString;
-        if ((newValue == null) || "".equals(newValue.toString()) || "NULL".equalsIgnoreCase(newValue.toString())) {
+        if ((newValue == null) || newValue.toString().isEmpty() || "NULL".equalsIgnoreCase(newValue.toString())) {
```

Comment:
Yeap, I didn't look for more of these occurrences because there are a bunch of them in the project and I was trying to limit the scope to just update the empty string check. I can update this one and also some other occurrences in the code if you think we should address that change in this PR.